# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/ptyprocess/hotfix-setup.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ptyprocess-0.7.0/setup.py.vanilla	2026-10-06 18:22:24.163855500 +0300
+++ ptyprocess-0.7.0/setup.py	2026-10-06 18:22:39.805921658 +0300
@@ -1,7 +1,7 @@
 #!/usr/bin/env python
 # setup.py generated by flit for tools that don't yet use PEP 517
 
-from distutils.core import setup
+from setuptools import setup
 
 packages = \
 ['ptyprocess']
